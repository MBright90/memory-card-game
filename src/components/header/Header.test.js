import Header from '@components/app'
import { render, screen } from '@testing-library/react'

test('renders learn react link', () => {
  render(<Header />)
  const logoElement = screen.getByText(/mind training/i)
  expect(logoElement).toBeInTheDocument()
})
