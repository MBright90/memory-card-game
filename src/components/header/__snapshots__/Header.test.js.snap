// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders to match snapshot 1`] = `
<div>
  <header
    class="container"
  >
    <div
      class="helpLink active"
      role="link"
      tabindex="-2"
    >
      <a
        href="#/"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-circle-info "
          data-icon="circle-info"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM216 336h24V272H216c-13.3 0-24-10.7-24-24s10.7-24 24-24h48c13.3 0 24 10.7 24 24v88h8c13.3 0 24 10.7 24 24s-10.7 24-24 24H216c-13.3 0-24-10.7-24-24s10.7-24 24-24zm40-208a32 32 0 1 1 0 64 32 32 0 1 1 0-64z"
            fill="currentColor"
          />
        </svg>
      </a>
    </div>
    <h1
      class="siteTitle"
    >
      MIND TRAINING
    </h1>
    <div
      class="scoreboard"
      role="presentation"
    >
      <p
        style="color: rgb(255, 255, 255);"
      >
        Score: 
        5
      </p>
      <p
        style="color: rgb(255, 255, 255);"
      >
        Best: 
        8
      </p>
    </div>
  </header>
</div>
`;
